setup.py

```diff
@@ -1,15 +1,23 @@
 # Copyright (C) Microsoft Corporation. All rights reserved.
 import setuptools
+import sys
 import os
 
-# Python 3 + build tools are required to install this library. To install the
-# prerequisites on Ubuntu, run
-# sudo apt-get install build-essential python3 python3-dev python3-venv
 
-version_file_path = os.path.abspath("topologic/version.txt")
-exec(open('topologic/version.py').read())
+def handle_version() -> str:
+    sys.path.insert(0, os.path.join("topologic", "version"))
+    from version import version
+    print(sys.path)
```

Comment:
Were these debugging prints or did you intend to keep them in here?

Reply:
They were debugging prints